tests/installation/test_installer_old.py

```diff
@@ -54,7 +54,7 @@ def load(
 
 class Locker(BaseLocker):
     def __init__(self, lock_path: str | Path) -> None:
-        self._lock = TOMLFile(Path(lock_path).joinpath("poetry.lock"))
+        self._lock = Path(lock_path.joinpath("poetry.lock"))
```

Comment:
This change was required, because there is no reason to keep `TOMFile` when `Locker._lock` is actually a `Path`.

No test is relying on `_lock` be a `TOMLFile`.